components/builder-web/app/sign-in-page/_sign-in-page.scss

```diff
@@ -15,6 +15,11 @@
 .hab-sign-in {
   @include shadow-float;
 
+  .mat-raised-button:not([class*=mat-elevation-z]) {
+    background-color: $hab-orange;
+    color: $hab-white;
+  }
+
```

Comment:
As you'll see below in the theme file, I changed the default accent from orange to blue which corrects all of the small form inputs/controls. In turn, I then had to override this call-to-action button to remain orange.